fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/version/AddVersionDAO.java

```diff
@@ -38,14 +38,18 @@ public AddVersionDAO(String adminSchemaName, String schemaName, String type, Str
 
     @Override
     public void run(IDatabaseTranslator translator, Connection c) {
-
-        final InsertStatement ins = InsertStatement.builder(adminSchemaName, SchemaConstants.VERSION_HISTORY)
+        InsertStatement.Builder insBuilder = InsertStatement.builder(adminSchemaName, SchemaConstants.VERSION_HISTORY)
                 .addColumn(SchemaConstants.SCHEMA_NAME)
                 .addColumn(SchemaConstants.OBJECT_TYPE)
                 .addColumn(SchemaConstants.OBJECT_NAME)
-                .addColumn(SchemaConstants.VERSION)
-                .addColumn(SchemaConstants.APPLIED, "CURRENT TIMESTAMP")
-                .build();
+                .addColumn(SchemaConstants.VERSION);
+        if (translator.getDriverClassName().contains("postgresql")) {
+            insBuilder = insBuilder.addColumn(SchemaConstants.APPLIED, "CURRENT_TIMESTAMP");
+        } else {
+            insBuilder = insBuilder.addColumn(SchemaConstants.APPLIED, "CURRENT TIMESTAMP");
+        }
+
+        final InsertStatement ins = insBuilder.build();
```

Comment:
If I could suggest a flip of the code here.  `if-then-else` can come before, and be assigned to a value.
then the builder can be automatically assigned to the InsertStatement

Reply:
make sense, done